cmd/manager/main.go

```diff
@@ -673,6 +678,9 @@ func setupWebhook(mgr manager.Manager, certRotation certificates.RotationParams,
 		}
 	}
 
+	// esv1 validating webhook is wired up differently, in  to access the k8s client
```

Comment:
Something seems missing here ?
```suggestion
	// esv1 validating webhook is wired up differently, in order to access the k8s client
```